Google.PowerShell/BigQuery/GcbqTable.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright 2015-2017 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Net;
+using System.Management.Automation;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Lists all tables in the specified dataset or returns a specific table.
+    /// </para>
+    /// <para type="description">
+    /// Lists all tables in the specified dataset. Requires the READER dataset role. If a table ID is specified, 
+    /// it will return the table resource, which describes the data in the table. Note that this is not the 
+    /// actual data from the table. If no Project is specified, the default project will be used. The dataset 
+    /// may be selected by passing in a Dataset object via pipeline or by passing the dataset ID with the 
+    /// -Dataset parameter. This cmdlet returns a TableList object if no Table was specified, and a Table otherwise.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Get-GcbqDataset “my_data” | Get-GcbqTable</code>
```

Comment:
The Microsoft [PowerShell Cmdlet guidlines](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx#SC02) encourage a pipeline input for every ParameterSet. Piping in the Dataset seems a reasonable use for this cmdlet, as it would let you get all tables for several Datasets (or even all tables with a specific name over several Datasets).